core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -118,10 +118,17 @@ private static Executor defaultBlockingTaskExecutor() {
     }
 
     private static final class DefaultBlockingTaskExecutorHolder {
-        static final Executor INSTANCE = new ThreadPoolExecutor(
-                0, DEFAULT_MAX_BLOCKING_TASK_THREADS,
-                60, TimeUnit.SECONDS, new LinkedTransferQueue<>(),
-                new DefaultThreadFactory("armeria-blocking-tasks", true));
+        static final Executor INSTANCE;
+
+        static {
+            final ThreadPoolExecutor instance = new ThreadPoolExecutor(
```

Comment:
Will do. :-) Also, the pool created by newCachedThreadPool() does not time out the thread that's once spawned IIRC.